lib/components/primitive-components/Group/Group.ts

```diff
@@ -898,29 +959,11 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
       }
     }
 
```

Comment:
all of this section can just be `return getPresetAutoroutingConfig(autorouter)` (even if autorouter is not defined, getPresetAutoroutingConfig can handle it)